modules/swagger-codegen-maven-plugin/README.md

```diff
@@ -45,7 +45,8 @@ mvn clean compile
 - `modelPackage` - the package to use for generated model objects/classes
 - `apiPackage` - the package to use for generated api objects/classes
 - `invokerPackage` - the package to use for the generated invoker objects
-- `modelNamePrefix` and `modelNameSuffix` - Sets the pre- or suffix for model classes and enums.
+- `modelNamePrefix` and `modelNameSuffix` - Sets the pre- or suffix for model classes and enums
+- `useJaxbAnnotations` - enable Jaxb annotations inside the generated models
```

Comment:
If I understand correctly, this feature is for `jaxrs-cxf` only so I suggest we document this feature is for `jaxrs-cxf` only for time being before we add similar feature for other JAX-RS generators.

(personally I prefer `enableJaxbAnnotations` but `useJaxbAnnotations` is also fine with me)